src/main/java/org/openstreetmap/atlas/generator/AtlasGeneratorHelper.java

```diff
@@ -560,6 +561,45 @@ public String getName()
         };
     }
 
+    protected static PairFunction<Tuple2<String, Atlas>, String, Atlas> subatlas(
+            final ConfiguredFilter filter, final AtlasCutType cutType)
+    {
+        return tuple ->
```

Comment:
Shouldn't this be cast as a Serializable function too so Spark can use it?
```suggestion
        return (Serializable & PairFunction<Tuple2<String, Atlas>, String, Atlas>) tuple ->
```

Reply:
Spark was able to use the function without the Serializable cast, but I can add it for safety!